### PR TITLE
Reorganize

### PR DESCRIPTION
**Pull request by TomAugspurger**
Reorganizing "Goals" and "Work Plan" to follow the instructions

1. Goals
  * The objective and expected outcomes of the proposal
  * A description of how the applicant will determine if the goals of the proposal have been successfully met
2. Work plan
  * A description of the proposed work the applicants are requesting funding for, including resources the applicants will provide that are not part of the requested funding. For software development related work (e.g., engineering, product design, user research), specify how the work fits into the existing project roadmap. For community outreach related activities (e.g., sprints, training), specify how these activities will be organized, the target audience, and expected outcomes

### CODE REVIEW

File: proposal.md [hunk 1]
```diff
@@ -130,13 +151,110 @@ foundational libraries in the Scientific Python Ecosystem including xarray,
...
+being part of numpydoc: https://github.com/numpy/numpydoc/issues/213.
+
+## Performance Monitoring
+
+This item does not require familiarity with pandas.
+
+Work can largely be broken into two buckets: maintenance and feature
+development. These two buckets can proceed in parallel.
+
+For developing a GitHub bot, we'll Heroku to respond to GitHub events.
```
Comment (jorisvandenbossche) [inline proposal.md]:
```suggestion
For developing a GitHub bot, we'll use Heroku to respond to GitHub events.
```

File: proposal.md [hunk 2]
```diff
@@ -130,13 +154,111 @@ foundational libraries in the Scientific Python Ecosystem including xarray,
...
+program][mentoring] to find people with the necessary skills and experience.
+
+## Extension Types
+
+This work item will require some familiarity with pandas' internals. We plan to
+more deeply integrate extension arrays into pandas, so that all columns in a
+`DataFrame` are backed by an `ExtensionArray`. This will simplify pandas
+internals and algorithms, improving the maintainability of the project.
+
+Additionally, changing every column to be backed by an `ExtensionARray` will
```
Comment (jorisvandenbossche) [inline proposal.md]:
```suggestion
Additionally, changing every column to be backed by an `ExtensionArray` will
```

**Comment by TomAugspurger** [comment]
I think I've incorporated most of the feedback from https://github.com/TomAugspurger/pandas-czi/pull/1 now.

**Comment by TomAugspurger** [comment]
After this, I think the biggest remaining items are

1. additional references to pandas (including biomedical-specific).
2. Figuring out timeline / effort levels of each item.

**Comment by TomAugspurger** [comment]
FYI, I think we're at ~1,900 words right now (max of 2,000 in the body)

**Comment by TomAugspurger** [comment]
Going to merge this so that the overall reorganization is in.

**Pull request merged**